src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml.cpp

```diff
@@ -49,3 +54,67 @@ void GraphingCalculator::ViewModel::set(GraphingCalculatorViewModel^ vm)
         RaisePropertyChanged(StringReference(sc_ViewModelPropertyName));
     }
 }
+
+void GraphingCalculator::GraphVariablesUpdated(Object^, Object^)
+{
+    m_viewModel->UpdateVariables(GraphingControl->Variables);
+}
+
+void GraphingCalculator::OnVariableChanged(Platform::Object^ sender, VariableChangedEventArgs args)
+{
+    GraphingControl->SetVariable(args.variableName, args.newValue);
+}
+
+
+void GraphingCalculator::SubmitTextbox(TextBox^ sender)
+{
+    auto variableViewModel = static_cast<VariableViewModel^>(sender->DataContext);
+
+    if (sender->Name == "Value")
```

Comment:
to be sure to not break this function if the XAML is modified:

`if(sender == Value)`
{
...
}`

The names of these TextBox should probably be changed to remove some confusions and conflicts, I don't expect this->Value to be a TextBox for example: 
proposal: `ValueTextBox`, `MinTextBox`, `MaxTextBox`, `StepTextBox`

Reply:
Done!